<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageStatusClass</class>
 <widget class="QWidget" name="ManageStatusClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>206</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ManageStatus</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="20,24,10,24,30,1,30,24,10,24,30,1,30,24,9,24,30,1,10,36,20,1,10,24,10,576,17">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>20</number>
     </property>
     <property name="rightMargin">
      <number>20</number>
     </property>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="start_icon">
         <property name="minimumSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image:url(:/img/res/manage/starttime.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="start_time">
         <property name="styleSheet">
          <string notr="true">color:#999999;margin:0 0 0 12;font:16px PingFang SC;</string>
         </property>
         <property name="text">
          <string>开始时间</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="begin_time">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>36</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>36</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;font:16px PingFang SC;color:#FFFFFF;margin:0 0 0 12;
QDateTimeEdit::bottom-button{border-image:url(:/img/res/manage/dwonarrow.png)};background:#444444;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="Line" name="line1">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>1</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>1</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background:#FFFFFF</string>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="stop_icon">
         <property name="minimumSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image:url(:/img/res/manage/stoptime.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="stop_time">
         <property name="styleSheet">
          <string notr="true">color:#999999;margin:0 0 0 12;font:16px PingFang SC;</string>
         </property>
         <property name="text">
          <string>结束时间</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="end_time">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>36</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>36</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;font:16px PingFang SC;color:#FFFFFF;margin:0 0 0 12;
QDateTimeEdit::bottom-button{border-image:url(:/img/res/manage/dwonarrow.png)};background:#444444;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_6">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="Line" name="line2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>1</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>1</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background:#FFFFFF</string>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_7">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="user_icon">
         <property name="minimumSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image:url(:/img/res/manage/userid.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="user_select">
         <property name="styleSheet">
          <string notr="true">color:#999999;margin:0 0 0 12;font:16px PingFang SC;</string>
         </property>
         <property name="text">
          <string>判图员</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_8">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="operator_box">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>36</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>36</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QComboBox{color:#FFFFFF;background:#444444;font:16px PingFang SC;margin:0 0 0 12;}</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_12">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="Line" name="line3">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>1</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>1</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background:#FFFFFF</string>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_9">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="pushButton">
       <property name="minimumSize">
        <size>
         <width>129</width>
         <height>36</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>129</width>
         <height>36</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton::hover{background:#666666;}QPushButton{ border-radius:8px; background:#444455; color:#FFFFFF; font:16px PingFang SC;}QPushButton::pressed{background:#555555;}</string>
       </property>
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_14">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_13">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="pac_icon">
         <property name="minimumSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image:url(:/img/res/manage/pacinfo.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="pac_info">
         <property name="styleSheet">
          <string notr="true">color:#999999;margin:0 0 0 12;font:16px PingFang SC;</string>
         </property>
         <property name="text">
          <string>包裹信息</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_10">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QListWidget" name="list_area">
       <property name="styleSheet">
        <string notr="true">QListWidget{color:rgb(200,200,200);border:3px solid gray;padding:20px 0px 0px 0px;font:16px PingFang SC;}
QListWidget::item{color:rgb(200,200,200);height:40px;padding-left:20px;border:0px solid gray;}
QListWidget::item:hover{color:rgb(255,255,255);}
QListWidget::item::selected:active{color:white;border-width:0;}
QListWidget::item:selected{color:white;border-width:0;}</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_11">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
